src/test/resources/hw1/CalculatorTestSuite.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Calculator suite">
+
+    <test name="Calculator test">
+        <classes>
+            <class name="hw1.CalculatorSubtractionTest"></class>
+            <class name="hw1.CalculatorMultiplyTest"></class>
+            <class name="hw1.CalculatorPowerOfDigitTest"></class>
+            <class name="hw1.CalculatorSineOfDigitTest"></class>
+            <class name="hw1.CalculatorIsNegativeTest"></class>
+            <class name="hw1.CalculatorDivideTest"></class>
+        </classes>
+    </test>
```

Comment:
Why do you decide include all list of classes in this suite? Do we have other way to create main suite for all tests?